libsrc/gamate/vga.inc

```diff
@@ -0,0 +1,228 @@
+
+; VGA charset for the PC-Engine conio implementation
+
+        .byte   $00, $00, $00, $00, $00, $00, $00, $00
+        ; 1
```

Comment:
to be honest, i dont quite remember where the "VGA" part comes from... iirc its from some old VGA Bios, but that could also be wrong :)